icu4c/source/common/unicode/unistr.h

```diff
@@ -343,7 +343,7 @@ class U_COMMON_API UnicodeString : public Replaceable
    * greater than the characters in `text`, false otherwise
    * @stable ICU 2.0
    */
-  inline UBool operator> (const UnicodeString& text) const;
+  inline bool operator> (const UnicodeString& text) const;
```

Comment:
We have discussed only changing `==` and `!=`. I don't remember discussing changing any other APIs/operators. Do we need to change these? (Looks like other than this unistr.h file everything else is just equality.)

Reply:
In order to just get the currently existing code to build as C++20, it's not necessary to change the return type of the relational operators now (the only reason I did it was that it seemed like a natural thing to do, making the code more consistent and paving the way for future C++20 work, such as the _spaceship_ operator `<=>`).

But it'd be trivial to revert if you think that'd be better.

Reply:
Please only change `==` and `!=` for now, matching the ticket and the discussion.

Reply:
I just read about the “spaceship operator” -- https://devblogs.microsoft.com/cppblog/simplify-your-code-with-rocket-science-c20s-spaceship-operator/

If you think that that will require us changing non-equality operators to return bool, then please submit a new ticket.

Reply:
> Please only change `==` and `!=` for now, matching the ticket and the discussion.

Done.